content/rules.md

```diff
@@ -10,6 +10,17 @@ These rules and the chosen format may change depending on the measures for the C
 
 # General Rules
 
+# Diversity Rules
+Each institution is permitted to send two teams by default. An institution can earn additional team slots under the following conditions:
+
+- The institution’s preliminaries consist of at least 30% female participants
+  - One additional slot awarded
+- The institution provides evidence of a genuine attempt to increase diversity at their preliminaries
+  - Maximum of 2 additional slots awarded
+  - The BAPC organizing committee determines the number of slots on a per-institution basis
+
+Coaches and teams participating in the BAPC are urged to read the official Code of Conduct \ref{CoC}. Discrimination of any kind will not be tolerated during the contest, and may be grounds for disqualification at the discretion of the Contest Director.
```

Comment:
For now I would remove the line about the Code of Conduct as its not there yet (and breaks the pipeline)

I agree that we should have that in the future.
```suggestion
Discrimination of any kind will not be tolerated during the contest, and may be grounds for disqualification at the discretion of the Contest Director.
```